### PR TITLE
Meteor integration

### PR DESCRIPTION
**Pull request by gilbertwat**
Hi @c-trimm!

I am a meteor user and found your library extremely useful :+1:  I have followed the [official~ish guide](https://github.com/MeteorCommunity/discussions/issues/14) to integrate. I didn't change the code of the library, but just the `gulpfile.js` and I put everything meteor related into `./meteor`. There are other packaging method but this can make sure the meteor package is up-to-date with your core library.

I have also open a thread on the meteor community to make this package official (here)[https://github.com/MeteorCommunity/discussions/issues/70]

Any thoughts on this?

### CODE REVIEW

File: .versions [hunk 1]
```diff
@@ -0,0 +1,4 @@
+gilbertwat:moment-recur@1.0.4
+meteor@1.1.6
+momentjs:moment@2.10.3
+underscore@1.0.3
```
Comment (bramski) [inline .versions]: I'm troubled to have multiple versions of dependency listings.  Can this file get auto-generated from the gulpfile & bower.json?

File: meteor/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+Packaging [moment-recur](https://github.com/c-trimm/moment-recur) for [Meteor.js](http://meteor.com) with the dependency on official [momentjs:moment](https://atmospherejs.com/momentjs/moment) package.
```
Comment (bramski) [inline meteor/README.md]: Is there any substance to this readme or is it just boilerplate?

File: meteor/package.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  api.versionsFrom(['METEOR@0.9.0', 'METEOR@1.0']);
+  api.use('momentjs:moment@2.10.2');
+  api.addFiles([
+      'meteor/export.js',
+      'moment-recur.js',
+      'meteor/clear.js'
+  ]);
+});
+
+/*
```
Comment (bramski) [inline meteor/package.js]: Why is this commented out?

File: package.json [hunk 4]
```diff
@@ -15,13 +15,15 @@
...
+    "gulp-uglifyjs": "~0.5.0",
+    "jasmine-core": "~2.1.3",
     "karma": "~0.12.31",
     "karma-chrome-launcher": "~0.1.7",
+    "karma-jasmine": "~0.3.4",
     "run-sequence": "~1.0.2",
-    "gulp-uglifyjs": "~0.5.0",
-    "jasmine-core": "~2.1.3",
-    "karma-jasmine": "~0.3.4"
+    "spacejam": "^1.1.4"
```
Comment (bramski) [inline package.json]: What is this for?

**Comment by bramski** [comment]
Ping.

**Comment by c-trimm** [comment]
No reply since August. Closing. @gilbertwat, if you'd like to reply and fix this up, please feel free to comment and I can reopen this PR.

**Pull request closed without merging**